ntbs-service/appsettings.json

```diff
@@ -12,7 +12,7 @@
     "ClientSecret": "<Provided by deployment secrets>",
     "Authority": "<Provided by deployment secrets>",
     "CallbackPath": "/signin-oidc",
-    "Enabled": false
+    "Enabled": true
```

Comment:
I've also taken this opportunity to make Azure AD the default auth - this won't really have any effect since all the environments are already using Azure AD.